build.gradle

```diff
@@ -845,7 +846,7 @@ project(':fmlonly') {
     }
 
     // Since we need the modules in the bootstrap, we need to make sure they are compiled before we do each run
-    afterEvaluate { prepareRuns.dependsOn(':fmlcore:jar', ':fmlloader:jar', ':javafmllanguage:jar', ':mclanguage:jar') }
+    afterEvaluate { prepareRuns.dependsOn(':fmlcore:jar', ':fmlloader:jar', ':javafmllanguage:jar', ':lowcodelanguage:jar', ':mclanguage:jar') }
```

Comment:
It looks to me like this is a duplication of PACKED_DEPS, so it should just reference PACKED_DEPS directly.

Reply:
This is how it is in existing code. Is this PR supposed to fix the build script for Forge as well? That sounds REALLY dumb.